test/test_torch.py

```diff
@@ -5538,6 +5538,19 @@ def test_abs(self):
         res = torch.LongTensor((-bignumber,))
         self.assertGreater(res.abs()[0], 0)
 
+    def test_hardshrink(self):
```

Comment:
We should delete the `hardshrink` in TH. After that, we can reuse the original hardshrink tests ([here](https://github.com/pytorch/pytorch/blob/28b592e00bcf7c1484964001817f1104e876e54c/test/test_nn.py#L5311-L5321)) so this new test is unneeded.